spec/compiler/crystal/tools/doc/project_info_spec.cr

```diff
@@ -92,23 +92,25 @@ describe Crystal::Doc::ProjectInfo do
 
       it "git non-tagged commit" do
         run_git "init"
+        run_git "checkout -b a_branch"
         run_git "add shard.yml"
         run_git "commit -m \"Initial commit\" --no-gpg-sign"
         commit_sha = `git rev-parse HEAD`.chomp
 
-        assert_with_defaults(ProjectInfo.new(nil, nil), ProjectInfo.new("foo", "master", refname: commit_sha))
+        assert_with_defaults(ProjectInfo.new(nil, nil), ProjectInfo.new("foo", "a_branch", refname: commit_sha))
```

Comment:
What's the reason for changing the tested branch name tho?

Reply:
`checkout -b master` will give an error if the branch already exist. `-B` may be an alternative to using a different branch name, but as I'm less certain how well any output from that is printed to terminal, this seemed easier.

Reply:
Output can simply be disabled with `--quiet`.

Reply:
Right, and it turned out it wasn't needed anyhow.